js/renovation/ui/scroll_view/__tests__/scrollview.test.tsx

```diff
@@ -0,0 +1,16 @@
+import {
+  ScrollView,
+} from '../scroll_view';
+
+describe('ScrollView', () => {
+  describe('Logic', () => {
+    describe('Getters', () => {
+      describe('cssClasses', () => {
+        it('should add dx-scrollview class by default', () => {
```

Comment:
Deep structure: Root\Logic\Getters\Property...
In test, we created a new ScrollView and checks values in its members.
Let's rename test to 'Check default property values'

Reply:
This structure uses for testing other widgets. Look at as a deal among our renovation teammates. 
The name I will fix in the next commit.

Reply:
Fixed in 6d6b02222985ba82b5cb0f0e957c13ec5d3b5a91